lib/awful/spawn.lua

```diff
@@ -370,6 +370,25 @@ function spawn.easy_async(cmd, callback)
     })
 end
 
+--- Call spawn.easy_async with a shell.
+-- This calls `cmd` with `$SHELL -c` (via `awful.util.shell`).
+-- @tparam string|table cmd The command.
+-- @function callback Function with the following arguments:
+--   @tparam string callback.stdout Output on stdout.
+--   @tparam string callback.stderr Output on stderr.
+--   @tparam string callback.exitreason Exit Reason.
+--   @tparam integer callback.exitcode Exit code.
+--   The exitreason argument can either be "exit" or "signal".
+--   For "exit" reason it's the exit code.
+--   For "signal" reason it's the signal causing process termination.
+-- @treturn[1] Integer the PID of the forked process.
+-- @treturn[2] string Error message.
+-- @see spawn.with_line_callback
+function spawn.easy_async_with_shell(cmd, callback)
+    local cmd = { util.shell, "-c", cmd or "" }
```

Comment:
you should remove `local`:
https://travis-ci.org/awesomeWM/awesome/jobs/199737866#L3492